### ISSUE TITLE
A Whole New Issue for the Project Checkpoint 3 Website Review

### ISSUE THREAD

**Issue by frabbitry** [lmcneil7/teen-titans#31]
@ebeshero @lmcneil7 @smdunn921 @amberpeddicord 
The guidelines for what should be done with the website for Project Checkpoint 3: 
Develop a page or pages for the project website, which use 
- An SSI navigation menu 
- Section, div, and span elements 
- CSS flex to control horizontal layout 
Furthermore, 
- Create a file directory structure for the website in the project GitHub repo. 

Since you guys already had a website, there wasn't much that needed to be done with the file directory structure. If I could make one suggestion to make your lives perhaps unnecessarily harder, I would say that you should probably make a separate folder for your Javascript files, instead of just throwing those in with the html files. Since Javascript may end up being very important for your eventual map section of the website, it might make your lives easier to separate out those files now, so that you know where all of that code is and where to put it (especially if a newbie like Amber starts working on the website).  

You're using an SSI on the map page, which checks that box. There are no divs, spans, or flexboxes, but I know that you know how to use those from the other parts of the website. The CSS also looks good. I definitely like the dark gray background. 

Eventually, you're going to want to start to think about putting maps on websites. Web mapping, somewhat unfortunately, can be an entirely different can of worms than mapping in general. 

I know that Lauren wanted to maybe use an actual map of San Francisco, which perhaps makes this project 100x harder but also 100x more interesting from a literary mapping-ish perspective (do we want to call Teen Titans literature? Do we want to go there?). You may want to go the KML route, in which case you'd look at displaying a Google Maps map and working with Google APIs. I don't know much about that, honestly (though if you guys were going to go that route, I'd definitely look into it). Alternatively, you could learn how to use GIS software like QGIS (I swear, it's really very easy to use). If you did use QGIS, I could show you guys how to get set up with QGIS cloud or how to use some of the QGIS plugins that generate web maps for you. With QGIS Cloud, you'd be using iframes to link to a web map that's hosted on the QGIS Cloud server. There's also the Javascript library Leaflet. Furthermore, a good resource to know about is [OpenStreetMap](https://www.openstreetmap.org/#map=5/38.007/-95.844) for geodata. 

Some cool websites that you could look at for inspiration might be [ORBIS](http://orbis.stanford.edu/) or any of the web maps linked to by [this](https://www.awwwards.com/20-inspirational-examples-of-interactive-maps-and-street-view-experiences-in-web-design.html) website. 

Of course, you could always choose the road more well traveled and do something more like what some of the video game projects ([Bloodborne](http://bloodborne.newtfire.org/), [Pokemon](http://pokemon.newtfire.org/), [Undertale](http://undertale.newtfire.org/)) have done and use a fictional map, with image mapping and iframes. 

In short, you guys have some options.

**Comment by smdunn921** [comment]
Thank you @frabbitry! I'll try to move the javascript files to a separate folder later tonight and get all that sorted out. I was going to add something last night, be it a div or whatnot but I fell asleep around 8... alas. I'm glad you like the dark grey background; a really cool person suggested we do that instead of black :eyes:. 

We can probably start talking about what we would want to do with maps at our meeting on monday? (@amberpeddicord @lmcneil7 :eyes: ) and spend the rest of the weekend looking at the sites you linked to get some ideas for then!
